.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
```

Comment:
4 space indentation ftw :(

Reply:
Not really, I used to say the same until I got used to it (airbnb guide). Code is more readable and uses much less columns. Give it a chance!